src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -130,11 +140,11 @@ public VerificationResult verifyPhoneWithCode(Request req, Response res) {
         if (code == null) {
             logMessageAndHalt(req, 400, "Missing code from verify request.");
         }
-        if (otpUser.pendingPhoneNumber == null) {
-            logMessageAndHalt(req, 404, "User must have valid phone number for SMS verification.");
+        if (StringUtils.isBlank(otpUser.phoneNumber)) {
```

Comment:
Is there a good reason to use `isBlank` instead of `isPhoneNumberValidE164`?

Reply:
It's evolution!

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/83/commits/8ec525aabdaac519033d9918bca89ad483ecaa7f.